### PR TITLE
Link requirements with Edraak's edx-ora2 v2.1.18

### PR DESCRIPTION
**Pull request by shadinaif**
Link requirements with Edraak's edx-ora2 v2.1.18

### CODE REVIEW

File: requirements/edx/github.in [hunk 1]
```diff
@@ -89,7 +89,6 @@
 # Our libraries:
 -e git+https://github.com/edx/codejail.git@a320d43ce6b9c93b17636b2491f724d9e433be47#egg=codejail
 -e git+https://github.com/edx/acid-block.git@e46f9cda8a03e121a00c7e347084d142d22ebfb7#egg=acid-xblock
--e git+https://github.com/edx/edx-ora2.git@2.1.17#egg=ora2==2.1.17
 -e git+https://github.com/edx/RecommenderXBlock.git@1.3.3#egg=recommender-xblock==1.3.3
```
Comment (ghost) [inline requirements/edx/github.in:96]:
```suggestion
# Edraak override
-e git+https://github.com/Edraak/edx-ora2.git@v2.1.18#egg=ora2==2.1.18

-e git+https://github.com/edx/RecommenderXBlock.git@1.3.3#egg=recommender-xblock==1.3.3
```

File: requirements/edx/github.in [hunk 2]
```diff
@@ -103,3 +102,6 @@
 -e git+https://github.com/mitodl/edx-sga.git@6b2f7aa2a18206023c8407e2c46f86d4b4c3ac96#egg=edx-sga==0.8.0
 -e git+https://github.com/open-craft/xblock-poll@add89e14558c30f3c8dc7431e5cd6536fff6d941#egg=xblock-poll==1.5.1
 -e git+https://github.com/edx-solutions/xblock-drag-and-drop-v2@v2.1.6#egg=xblock-drag-and-drop-v2==2.1.6
+
+# Edraak override
```
Comment (ghost) [inline requirements/edx/github.in]:
```suggestion
```

File: requirements/edx/github.in [hunk 3]
```diff
@@ -103,3 +102,6 @@
...
+-e git+https://github.com/Edraak/edx-ora2.git@v2.1.18#egg=ora2==2.1.18
```
Comment (ghost) [inline requirements/edx/github.in]:
```suggestion
```

File: requirements/edx/github.in (hunk 1 above)
Comment (ghost) [reply requirements/edx/github.in:96]: Please override on the same line. It makes git conflict resolution easier to reason about.

**Review by ghost — CHANGES_REQUESTED** [review]
Please check the suggestions.

File: requirements/edx/github.in (hunk 1 above)
Comment (shadinaif) [reply requirements/edx/github.in:96]: Done

Comment (shadinaif) [reply requirements/edx/github.in:96]: Done

File: requirements/edx/github.in (hunk 3 above)
Comment (shadinaif) [reply requirements/edx/github.in]: Done

**Comment by shadinaif** [comment]
CI Failed, but **19 Failed Tests** is our Base for now

**Pull request merged**